install/gcp/installer/forseti_client_installer.py

```diff
@@ -49,14 +49,15 @@ def deploy(self, deployment_tpl_path, conf_file_path, bucket_name):
             bool: Whether or not the deployment was successful
             str: Deployment name
         """
+        gcloud.grant_client_svc_acct_roles(
```

Comment:
Moving this up will alleviate the race condition, but will not guarantee that it is solved in a deterministic way.

What we really want to do, is to verify after the roles are granted, that the permissions are indeed enabled on the platform, before doing the subsequent deploy steps.

It's not clear what we can do to deterministically verify the permissions are indeed enabled.  Use gcloud to do a list iam policy?  

Or should we set a reasonable pause of N seconds?